src/Stratis.Bitcoin.IntegrationTests/Miners/ProofOfStakeMintCoinsSpecification.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Stratis.Bitcoin.IntegrationTests.TestFramework;
+using Xunit;
+
+namespace Stratis.Bitcoin.IntegrationTests.Miners
+{
+    public partial class ProofOfStakeMintCoinsSpecification : BddSpecification
+    {
+        [Fact]
+        public void Staking_wallet_can_mint_new_coins()
+        {
+            Given(a_proof_of_work_node_with_wallet);
+            And(it_mines_genesis_and_premine_blocks);
+            And(mine_coins_to_maturity);
+            And(a_proof_of_stake_node_with_wallet);
+            And(it_syncs_with_proof_work_node);
+            And(create_tx_to_send_million_coins_from_pow_wallet_to_pos_node_wallet);
+            And(pow_wallet_broadcasts_tx_of_million_coins_and_pos_wallet_receives);
+            And(pos_node_mines_a_further_ten_blocks);
+            When(pos_node_starts_staking);
+            Then(pos_node_wallet_has_earned_coins_through_staking);
+        }
```

Comment:
Overall, I like the detail - perhaps some tense standardisation and perhaps this is overgranular - but at this stage with us all new to this expressive style, being overgranular is probably not a bad thing at this stage.

As a fairly new developer to the codebase, seeing  this high level overview of the test at this level immediately gives me some understanding of what is going on.  And to be able to drill in and see the details of how these things are achieved is really nice when it is compartmentalised like this.  
@ferdeen @monsieurleberre @maciejzaleski , as new developers, do you find this easy to read and get the higher level bigger picture of what this test (and that part of the system) is about?

In many ways, I don't need to drill in, (once we know the inside is accurate) - reading the integration test steps alone starts to give me the bigger picture of the system.